src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/1.sql

```diff
@@ -1,43 +1,332 @@
+-- NOTE: This script DROPS AND RECREATES all database objects.
+-- Style guide: please see: https://github.com/ktaranov/sqlserver-kit/blob/master/SQL%20Server%20Name%20Convention%20and%20T-SQL%20Programming%20Style.md
+
+
+/*************************************************************
+    Drop existing objects
+**************************************************************/
+
+DECLARE @sql nvarchar(max) =''
+
+SELECT @sql = @sql + 'DROP PROCEDURE ' + name + '; '
+FROM sys.procedures
+
+SELECT @sql = @sql + 'DROP TABLE ' + name + '; '
+FROM sys.tables
+
+SELECT @sql = @sql + 'DROP TYPE ' + name + '; '
+FROM sys.table_types
+
+SELECT @sql = @sql + 'DROP SEQUENCE ' + name + '; '
+FROM sys.sequences
+
+EXEC(@sql)
+
+GO
+
+/*************************************************************
+    Configure database
+**************************************************************/
+
 -- Enable RCSI
 ALTER DATABASE CURRENT SET READ_COMMITTED_SNAPSHOT ON
-GO
 
-IF NOT EXISTS(SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = 'SchemaVersion' AND TABLE_SCHEMA = 'dbo')
-BEGIN
-    CREATE TABLE dbo.SchemaVersion (
-        [Version] int PRIMARY KEY, 
-        [Status] varchar(10) 
-    )
-END
+-- Avoid blocking queries when statistics need to be rebuilt
+ALTER DATABASE CURRENT SET AUTO_UPDATE_STATISTICS_ASYNC ON
 GO
 
+/*************************************************************
+    Schema bootstrap
+**************************************************************/
+
+CREATE TABLE dbo.SchemaVersion
```

Comment:
Should we include a date as well?  For upgrades, it might be useful to track when they happened.

Reply:
Good idea. @brandonpollett, want to add that?